StacheMan/stachespace
# Site refactor

# Contributions

* Site style refactor
* Adds mobile support

# Testing

1. Load app into browser
2. Launch dev tools and go into responsive mode
3. Resize window and observe that content stays readable
